config-model/src/test/java/com/yahoo/vespa/model/application/validation/change/RedundancyChangeValidatorTest.java

```diff
@@ -0,0 +1,65 @@
+package com.yahoo.vespa.model.application.validation.change;
```

Comment:
Please add copyright.

Reply:
I have a script doing that for all files missing it and I hereby commit to doing another round of it soon.